seatunnel-connectors-v2/connector-http/connector-http-wechat/src/main/java/org/apache/seatunnel/connectors/seatunnel/wechat/sink/WeChatSink.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.seatunnel.connectors.seatunnel.wechat.sink;
+
+import org.apache.seatunnel.api.common.PrepareFailException;
+import org.apache.seatunnel.api.sink.SeaTunnelSink;
+import org.apache.seatunnel.api.sink.SinkWriter;
+import org.apache.seatunnel.api.table.type.SeaTunnelRow;
+import org.apache.seatunnel.common.constants.PluginType;
+import org.apache.seatunnel.connectors.seatunnel.common.sink.AbstractSinkWriter;
+import org.apache.seatunnel.connectors.seatunnel.http.config.HttpParameter;
+import org.apache.seatunnel.connectors.seatunnel.http.sink.HttpSink;
+
+import org.apache.seatunnel.shade.com.typesafe.config.Config;
+
+import com.google.auto.service.AutoService;
+
+@AutoService(SeaTunnelSink.class)
+public class WeChatSink extends HttpSink {
+
+    @Override
+    public String getPluginName() {
+        return "WeChat";
+    }
+
+    private final HttpParameter httpParameter = new HttpParameter();
+
+    private final String webHookUrl = "url";
+
+    @Override
+    public void prepare(Config pluginConfig) throws PrepareFailException {
```

Comment:
This overwrite method is useless, because httpSink had checked the parameter `url`.

Reply:
this method use to construct httpParameter which WeChatHttpSinkWriter constructor needed.
WeChatHttpSinkWriter redefind a  SeaTunnelRowType for WeChat send content in constructor.

Reply:
In my option, you can change `httpParameter` that in connector-http-base from `private` to `protected`.

Reply:
> In my option, you can change `httpParameter` that in connector-http-base from `private` to `protected`.

Thx,i will try .this way is  more convenient.